eden/util/display.py

```diff
@@ -272,9 +272,14 @@ def draw_graph_set(graphs,
                    n_graphs_per_line=5,
                    size=4,
                    edge_label=None,
+                   pos=None,
                    **args):
     """draw_graph_set."""
     graphs = list(graphs)
+    if pos:
+        for g, p in zip(graphs, pos):
+            g.graph['positiondictionary'] = p
```

Comment:
Alternatively, `[g.graph['positiondictionary'] = p for g, p in zip(graphs, pos)]`